metron-interface/metron-config/angular-cli.json

```diff
@@ -0,0 +1,51 @@
+{
+  "project": {
+    "version": "1.0.0-beta.15",
```

Comment:
This is the angular-cli version not the application version, the application version is mentioned in package.json. This is auto generated by angular-cli.